.gitignore

```diff
@@ -1,10 +1,8 @@
-.idea
```

Comment:
Is there a reason `.idea` is removed?

Reply:
`.idea` is not a project specific folder. This should pattern belongs in your global ignore file.

Reply:
`.idea` should be in `gitignore`, its a product of most jetbrains IDEs, and has no place on git. Suggest resolving this review

Reply:
> has no place on git. 

Correct. Which is why is belongs in the [global gitignore file](https://docs.github.com/en/github/getting-started-with-github/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer).

Reply:
Apologies I was a bit too brief there (just got home). 

I agree with it being in global, however for the sake of making sure *everybody* who contributes to this not accidentally committing `.idea` it makes sense to have it in a project specific gitignore as well. There is no issue with duplicate entries in project/global afaik (please do correct if im wrong there)

Imo its best to keep the barrier of entry low, and not everyone knows about global config for git (let alone how to use `git config` 😆), hence why i think this should just be resolved rather than accepted

Reply:
I think this should be removed, it belongs in the global gitignore file, and if people don't know what that is and accidently commit that all prs should be checked anyways

Reply:
I strongly feel that `.idea` should be left out as it's not hard to setup a global gitignore. Also as @AnotherCat has said, it would be caught in code review.